rakefile_common.rb

```diff
@@ -156,6 +156,15 @@ def instructions(key)
   @instructions[key]
 end
 
+def sources
+  @sources ||= begin
```

Comment:
Oops, yep, fixed in https://github.com/everypolitician/everypolitician-data/pull/7014/commits/5b6c13bb6eb4f62ff4385fa819edd5b7f80eb8b5.